src/EditorFeatures/CSharpTest/Diagnostics/GenerateEnumMember/GenerateEnumMemberTests.cs

```diff
@@ -1785,7 +1785,7 @@ void Main()
 enum Color : ulong
 {
     Green = 1UL << 0,
-    Blue = 1UL << 1
+    Blue = 1UL << 1,
```

Comment:
this appears to eb a regression in all these tests.  i would expect nothing to change unless the test explicitly turned this option on.

Reply:
This is impacted by the ongoing discussion in https://github.com/dotnet/roslyn/pull/54859/files#r671040273